rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1039,7 +1038,7 @@ void SPUThread::set_interrupt_status(bool enable)
 
 u32 SPUThread::get_ch_count(u32 ch)
 {
-	LOG_TRACE(SPU, "get_ch_count(ch=%d [%s])", ch, ch < 128 ? spu_ch_name[ch] : "???");
+	LOG_TRACE(SPU, "get_ch_count(ch=%d [%s])", ch, spu_ch_name[ch]);
```

Comment:
Range check here is necessary for correctness. Optimizer should move the check anyway when the trace is disabled. If this doesn't happen though, there is another trick...